docs/UserGuide.md

```diff
@@ -82,26 +88,42 @@ The quick start section provides a tutorial which will guide you through the ste
 
 <br>
 
-1. Ensure you have Java `11` or above installed in your Computer.
+<p><b>Step 1.</b></p>
+
+Ensure you have Java `11` or above installed in your Computer.
    * To check if you have this installed or for instructions on how to install, refer to our FAQ [here](#frequently-asked-questions).
 
+<br>
+
+<p><b>Step 2.</b></p>
 
-1. Download the latest `clientbook.jar` [here](https://github.com/AY2021S2-CS2103T-W15-2/tp/releases/tag/v1.2).
+Download the latest `clientbook.jar` [here](https://github.com/AY2021S2-CS2103T-W15-2/tp/releases/tag/v1.4).
    
+<br>
+
+<p><b>Step 3.</b></p>
 
-1. Copy the file to the folder (we will be referring to this folder as the _home folder_) where you want to store the ClientBook application and your client information.
+Copy the file to the folder (we will be referring to this folder as the **_home folder_**) where you want to store the ClientBook application and your client information.
 
+<br>
 
-1. Double-click the downloaded `clientbook.jar` file in your home folder to start the app. The screen below should appear in a few seconds. 
+<p><b>Step 4.</b></p>
+
+Double-click the downloaded `clientbook.jar` file in your home folder to start the app. The screen below should appear in a few seconds. 
    * Alternatively, for those who are tech-savvy, you can launch the application via Command Prompt in Windows or Terminal in Mac OS/Linux using the following command: `java -jar clientbook.jar`. 
```

Comment:
Should we have a tutorial on how to launch the terminal applications in the current directory? In the event that someone is not tech-savvy, but uses the lock functionality such that they are forced to unlock the application through the terminal.

Reply:
Sure it will be helpful in the case where someone who is not tech-savy actually uses our application but there won't be such a scenario because the only people who will be using this are the PE testers.